charts/argo-cd/values.yaml

```diff
@@ -371,6 +371,9 @@ configs:
   # SSH known hosts for Git repositories
   ## Ref: https://argo-cd.readthedocs.io/en/stable/operator-manual/declarative-setup/#ssh-known-host-public-keys
   ssh:
+    # -- Specifies if the argocd-ssh-known-hosts-cm configmap should be created by Helm.
+    enabled: true
```

Comment:
(Question zo other maintainers):
Do we wanna use `create` as a keyword here to be inline with other ConfigMaps conditionals?

/cc: @yu-croco , @tico24 , @mbevc1

Reply:
I vote to `create` to align existing ones. 😀

Reply:
I concur with @yu-croco

Reply:
changed it to `create`